src/main/game/PlayerFactory.java

```diff
@@ -0,0 +1,34 @@
+package main.game;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static main.game.PlayerTypes.O_MARK;
+import static main.game.PlayerTypes.X_MARK;
+
+public class PlayerFactory {
+
+    private final UI ui;
+    private final String playerOne;
+    private final String playerTwo;
+
+    public PlayerFactory(UI ui) {
+        this.ui = ui;
+        this.playerOne = X_MARK.getPlayer();
+        this.playerTwo = O_MARK.getPlayer();
+    }
+
+    public List<Player> getPlayerTypes(String players) {
+        Player[] humanVsHuman = {new HumanPlayer(ui, playerOne), new HumanPlayer(ui, playerTwo)};
+        Player[] humanVsComputer = {new HumanPlayer(ui, playerOne), new Computer()};
+
+        Map<String, List<Player>> playerTypes = new HashMap<>();
```

Comment:
I don't really understand why you create this HashMap. Couldn't you just have an if/else statement and return humanVsHuman or humanVsComputer depending on what the user chooses? Maybe I'm missing something here...